### ISSUE TITLE
Undefined results when multiple wxWidgets installations available (win32)

### ISSUE THREAD

**Issue by leamas** [Rasbats/shipdriver_pi#228]
As reported by @transmitterdan: https://github.com/OpenCPN/plugins/issues/468#issuecomment-947677873

**Comment by transmitterdan** [comment]
>> hm... Does it have to be this way?

No, but it is a PITA to have multiple git repositories of the same github project which is the only alternative I can see.  There are many different library flavors that can be created by the wx build procedures (x64, win32, debug, non-debug, uni-code, non-unicode, etc.) and they all live in separate trees under wxWidgets\lib.  So clearly this is what the wx people intended.

I think cmake is ignoring the environment variable but I have no idea why.

The file setup.h appears in every lib folder because the wx build process copies it there.  There are lots of them in a typical wx lib tree.  I don't understand why cmake just digs through the lib tree looking for the first setup.h it finds.  That is a bug in my view.  I have never had much luck discussing things like this with Canonical.  It is a non-issue if there is only one library tree below wxWidgets/lib so for most people it never arises as an issue.

I am not suggesting we worry about this issue with our CI builds.  I am only interested in this when locally building a debug build of wxWidgets and OpenCPN.  That is something we never do via CI AFAIK.
